src/main/java/com/practicesoftwaretesting/cart/model/Product.java

```diff
@@ -0,0 +1,21 @@
+package com.practicesoftwaretesting.cart.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class Product{
```

Comment:
format your code, please
```suggestion
public class Product {
```

Reply:
Formatted